bus/client.go

```diff
@@ -638,7 +638,7 @@ func (c *Client) SearchObjects(ctx context.Context, bucket, key string, offset,
 	return
 }
 
-func (c *Client) Object(ctx context.Context, path string, opts ...api.ObjectsOption) (o api.Object, entries []api.ObjectMetadata, err error) {
+func (c *Client) Object(ctx context.Context, path string, opts ...api.ObjectsOption) (o api.Object, entries []api.ObjectMetadata, hasMore bool, err error) {
```

Comment:
Hm ok, our API was already returning an objects response so I guess it makes sense to have the client simply match the response type and not explode it into various return params. I wasn't a big fan initially though because it's a bit of an odd endpoint with the object and the entries being either-or, so often you good easily ignore part of the response if you were certain for instance you were expecting entries.